src/criminisi.jl

```diff
@@ -24,8 +24,8 @@ struct Criminisi <: InpaintAlgo
 end
 
 # implementation follows the notation in the paper
-function inpaint_impl(img::AbstractArray{T,2}, mask::AbstractArray{Bool,2},
-                      algo::Criminisi) where {T}
+function inpaint_impl(img::AbstractArray{<:AbstractGray}, mask::AbstractArray{Bool,2},
+        algo::Criminisi)
```

Comment:
Please consider fixing the TAB in your editor to a fixed number of white spaces.